test/README.md

```diff
@@ -0,0 +1,14 @@
+# Testing
+
+## Schema Generation
+
+To add/update a test case for generating a valid schema from a typscript file:
+
+- Look in `test/valid-data` for a sample related to your change. If you don't find one, create your own following the naming convention. For example, when adding the new sample `annotation-default`:
+    - Create folder `test/valid-data/annotation-default`
+    - Add `main.ts` to that folder with the type sample
+- Update the corresponding `main.ts` file with your changes.
+- Compile the JSON Schema via `yarn --silent run run --path 'test/valid-data/{the-target-folder}/*.ts' --type 'MyObject' -o test/valid-data/{the-target-folder}/schema.json`
+    - This happens automatically when running `yarn test:update`, but that can take a while
```

Comment:
I think I still prefer

```suggestion
- Run `yarn test:update` to compile the JSON schema
```

since it automatically uses the same compile options the test uses. Just makes things easier (albeit slower).